packages/ui/components/validate-messages/translations/de.js

```diff
@@ -5,13 +5,13 @@ export default {
     MinLength: 'Du musst mindestens {params} Zeichen eingeben.',
     MaxLength: 'Du kannst maximal {params} Zeichen eingeben.',
     MinMaxLength: 'Du musst zwischen {params.min} und {params.max} Zeichen eingeben.',
-    Pattern: 'Geben Sie ein gültiges {fieldName} ein.',
-    IsNumber: 'Geben Sie ein gültiges {fieldName} ein.',
+    Pattern: 'Geben Sie ein gültige {fieldName} ein.',
+    IsNumber: 'Geben Sie ein gültige {fieldName} ein.',
     MinNumber: 'Geben Sie für {fieldName} einen Wert über {params} ein.',
     MaxNumber: 'Geben Sie für {fieldName} einen Wert unter {params} ein.',
     MinMaxNumber:
       'Geben Sie für {fieldName} einen Wert zwischen {params.min} und {params.max} ein.',
-    IsDate: 'Bitte geben Sie ein gültiges Datum ein (TT.MM.JJJJ).',
+    IsDate: 'Bitte geben Sie ein gültige Datum ein (TT.MM.JJJJ).',
```

Comment:
here 'gültiges' is correct. 
```suggestion
    IsDate: 'Bitte geben Sie ein gültiges Datum ein (TT.MM.JJJJ).',
```